enos/modules/target/main.tf

```diff
@@ -50,7 +44,7 @@ resource "aws_instance" "target" {
   ami                    = var.ami_id
   instance_type          = var.instance_type
   vpc_security_group_ids = [aws_security_group.boundary_target.id]
-  subnet_id              = tolist(data.aws_subnets.infra.ids)[count.index % length(data.aws_subnets.infra.ids)]
+  subnet_id              = var.subnet_ids[count.index % length(var.subnet_ids)]
```

Comment:
Clarification question: I don't totally follow this. So, from the boundary cluster side, how come there are multiple subnet_ids? Why does it change based on the count index?

Reply:
This is to allow multiple targets, what we get from the infra module is a list of every subnet available to create in, and normally, we would want to distribute instances across them to test networking boundaries. So if you spun up 4 instances in an AZ that had 4 subnets available, we'd put once instance in each, instead of all 4 in one